src/storage/mvcc/txn.rs

```diff
@@ -161,20 +161,25 @@ impl<S: Snapshot> MvccTxn<S> {
         }
     }
 
+    /// Checks whether the existence of the key is according to `should_not_exist`.
```

Comment:
It now looks weird. Maybe `Checks whether the key exists according...` or `Checks the existence of the key according...` would be better